### ISSUE TITLE
Tiled galleries: editing alt text replaces unrelated images on the same page.

### ISSUE THREAD

**Issue by ariel-maidana** [Automattic/wp-calypso#76425]
### Quick summary

Editing the alt text of an image that is part of a tiled gallery results in other (unrelated) images on the same page being replaced.

### Steps to reproduce

1. Create a page with at least a Tiled Gallery block and several other Image blocks.
2. Select the Tiled Gallery block.
3. Click on the edit icon on the Tiled Gallery block toolbar.
4. Click on "Edit"
5. Modify the alt text
6. Click on "Insert".

### What you expected to happen

Only the alt text of the image we touched should be modified.
Other blocks should not be affected.

### What actually happened

Several other images on the page were replaced by the image in which we replaced the alt text.

### Impact

One

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Atomic

### Logs or notes

- I was able to reproduce the issue on a test site using Firefox.
- Reported in 6214915-zd-woothemes

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple – Could Not Replicate
- Tested on Atomic – Could Not Replicate

📌 FINDINGS/SCREENSHOTS/VIDEO
- Tested this with a 4-image Tiled Gallery, followed by 2 additional image blocks.  Updating the Alt Text in the Tiled gallery worked as expected, in both Simple and Atomic.
- Tested on theme Twenty Twenty-Three, WordPress 6.2, Gutenberg 15.6.2

#### Screenshots/Recordings

https://user-images.githubusercontent.com/27249804/235740162-16db9294-2dcc-4cb5-87a0-da800fec912f.mov

📌 ACTIONS
- Requested author feedback

📌 Message to Author
- Hey @ariel-maidana ! Can you take a look at my recording above and let me know if we're taking any different steps in our tests?  I see you were able to reproduce the issue in a test site - was that using a copy of the user's content, or your own?  
- Any additional details you can spot/provide here would be much appreciated, thank you!

**Comment by ariel-maidana** [comment]
Hi @cuemarie!  Thank you for looking in to this!

> Can you take a look at my recording above and let me know if we're taking any different steps in our tests? 

The steps are the same.

> I see you were able to reproduce the issue in a test site - was that using a copy of the user's content, or your own?

I copied the page contents from the user's site.

> Any additional details you can spot/provide here would be much appreciated, thank you!

The affected site is using Hever.

**Comment by cuemarie** [comment]
Thanks for these details @ariel-maidana . Unfortunately, I'm still not able to replicate this user's experience.  I've reviewed the youtube video they shared on the ticket, as well as their site.  I've tried the following:

- Activated Hever on a Personal Plan site
- Used a page for testing, to match the user's site
- Created a Group block with nested tile block and image blocks to match the user's set up
- Duplicated that group twice, and replaced the images within it, in case duplicating the gallery causes weird behavior
- Made the settings on my Settings > Media page match the user's set up 
     - ![OTQyoF.png](https://user-images.githubusercontent.com/27249804/236348963-2f2e55d2-5442-49b0-b26d-9dd699379640.png)

https://user-images.githubusercontent.com/27249804/236348869-8bc5d72f-4b93-4c15-abf9-53d0d2bfe69a.mp4

In each test, the tiled gallery allows me to update my ALT text without replicating the issue.

**Next Steps**
- At this point, I think this would be best addressed with an internal report on 2EDhh-p2. 
- We might also ask the user to delete the tiled gallery that is triggering that issue and rebuild it from scratch. Their page itself was originally published in December 2021 - perhaps updating the block itself may resolve the issue.

📌 ACTIONS
- Closed as not reproducible
